src/ontology/mondo-edit.obo

```diff
@@ -434328,8 +434325,10 @@ xref: MEDGEN:502456 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:77295 {source="MONDO:equivalentTo"}
 xref: SCTID:722064003 {source="MONDO:equivalentTo"}
 xref: UMLS:C3502054 {source="MEDGEN:502456", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
+is_a: MONDO:0011897 {source="https://orcid.org/0000-0001-5493-2602"} ! leukodystrophy, hypomyelinating, 7, with or without oligodontia and/or hypogonadotropic hypogonadism
 is_a: MONDO:0019046 {source="Orphanet:77295"} ! leukodystrophy
-relationship: excluded_subClassOf MONDO:0011897 {source="https://orcid.org/0000-0001-5208-3432"} ! leukoencephalopathy-ataxia-hypodontia-hypomyelination syndrome
+relationship: excluded_subClassOf MONDO:0011897 {source="https://orcid.org/0000-0001-5208-3432"} ! leukodystrophy, hypomyelinating, 7, with or without oligodontia and/or hypogonadotropic hypogonadism
```

Comment:
Same as above, both SC and excluded SC. This requires also an additional source

Reply:
I removed the excluded_subClassOf axiom